library/network_connections.py

```diff
@@ -1029,14 +1029,17 @@ def connection_create(self, connections, idx, connection_current=None):
             addrs4 = list([a for a in ip["address"] if a["family"] == socket.AF_INET])
             addrs6 = list([a for a in ip["address"] if a["family"] == socket.AF_INET6])
 
+            is_ipv4_available = False
             if ip["dhcp4"]:
                 s_ip4.set_property(NM.SETTING_IP_CONFIG_METHOD, "auto")
                 s_ip4.set_property(
                     NM.SETTING_IP_CONFIG_DHCP_SEND_HOSTNAME,
                     ip["dhcp4_send_hostname"] is not False,
                 )
+                is_ipv4_available = True
             elif addrs4:
                 s_ip4.set_property(NM.SETTING_IP_CONFIG_METHOD, "manual")
+                is_ipv4_available = True
             else:
```

Comment:
What do you think about `is_ipv4_configured`? The current variable seems confusing to me.

Reply:
Updated. thanks.